src/scripts/main.js

```diff
@@ -357,4 +357,41 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+function getGender(gender) {
+  if (gender === 'm') {
+    return 'Male';
+  }
+
+  if (gender === 'f') {
+    return 'Female';
+  }
+}
+
+const table = document.querySelector('tbody');
+
+for (let i = 0; i < people.length; i++) {
+  const item = document.createElement('tr');
+
+  table.append(item);
+
+  const cellName1 = document.createElement('td');
+  const cellName2 = document.createElement('td');
```

Comment:
```suggestion
  const cellGender = document.createElement('td');
```
The previous comment meant not literally `cellName` for every cell, but replace the current variable name with a descriptive cell name